src/app/portlets/dot-edit-page/layout/components/dot-edit-layout-sidebar/dot-edit-layout-sidebar.component.html

```diff
@@ -0,0 +1,4 @@
+<div>
+    <h6 [ngClass]="'dot-edit-layout-sidebar--' + value.location">Sidebar</h6>
```

Comment:
this text have to by request to the BE

Reply:
In fact... I didn't check it first, but we do have keys for header, footer and sidebar in the language.properties, please use those Alfredo.

Reply:
done